sample_files/.travis.yml

```diff
@@ -6,7 +6,7 @@ cache:
     - $HOME/.cache/pip
 
 python:
-  - "2.7"
+  - "2.7.13"
```

Comment:
But why 2.7.13 and not 2.7.14? 🤔

Reply:
because that's what both travis' and runbot's docker images support currently